### PR TITLE
Tweak: Widgets search accepts non-english strings [ED-3145]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: assets/dev/js/editor/regions/panel/pages/elements/views/search.js [hunk 1]
```diff
@@ -3,14 +3,16 @@ var PanelElementsSearchView;
...
+
 	id: 'elementor-panel-elements-search-wrapper',
 
 	ui: {
 		input: 'input',
 	},
 
 	events: {
-		'input @ui.input': 'onInputChanged',
+		'keyup @ui.input': 'onInputChanged',
```
Comment (matipojo) [inline assets/dev/js/editor/regions/panel/pages/elements/views/search.js:15]: Try to paste, instead of typing.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/regions/panel/pages/elements/views/search.js (hunk 1 above)
Comment (StyleShit) [reply assets/dev/js/editor/regions/panel/pages/elements/views/search.js:15]: See #15056

Comment (matipojo) [reply assets/dev/js/editor/regions/panel/pages/elements/views/search.js:15]: Try with the mouse context menu.

Comment (StyleShit) [reply assets/dev/js/editor/regions/panel/pages/elements/views/search.js:15]: @matipojo 
See #15062

**Review by matipojo — APPROVED** [review]

**Pull request merged**